### PR TITLE
[Spotless] Applying Google Code Format for sql files #11

### PR DESCRIPTION
**Pull request by MitchellGale**
### Description
Applies spotless for SQL directory in project. Sets checkstyle to not fail on failure for sql directory.
 
### Issues Resolved
https://github.com/opensearch-project/sql/issues/1101
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [x] New functionality has been documented.
  - [x] New functionality has javadoc added
  - [x] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: sql/src/main/java/org/opensearch/sql/sql/parser/AstSortBuilder.java [hunk 1]
```diff
@@ -57,8 +54,8 @@ private List<Field> createSortFields() {
   }
 
   /**
-   * Argument "asc" is required.
-   * Argument "nullFirst" is optional and determined by Analyzer later if absent.
+   * Argument "asc" is required. Argument "nullFirst" is optional and determined by Analyzer later
```
Comment (Yury-Fridlyand) [inline sql/src/main/java/org/opensearch/sql/sql/parser/AstSortBuilder.java:57]: I think this comment is outdated and can be removed.

**Review by Yury-Fridlyand — DISMISSED** [review]

**Review by Yury-Fridlyand — APPROVED** [review]

File: sql/src/main/java/org/opensearch/sql/sql/parser/AstSortBuilder.java (hunk 1 above)
Comment (MitchellGale) [reply sql/src/main/java/org/opensearch/sql/sql/parser/AstSortBuilder.java:57]: Are you addressing the entire comment?

File: sql/src/test/java/org/opensearch/sql/sql/antlr/HighlightTest.java [hunk 2]
```diff
@@ -15,14 +15,14 @@ void single_field_test() {
...
+    acceptQuery(
+        "SELECT HIGHLIGHT(Tags), HIGHLIGHT(Body) FROM Index "
+            + "WHERE MULTI_MATCH([Tags, Body], 'Time')");
   }
 
   @Test
   void wildcard_test() {
-    acceptQuery("SELECT HIGHLIGHT('T*') FROM Index "
-        + "WHERE MULTI_MATCH([Tags, Body], 'Time')");
+    acceptQuery("SELECT HIGHLIGHT('T*') FROM Index " + "WHERE MULTI_MATCH([Tags, Body], 'Time')");
```
Comment (GumpacG) [inline sql/src/test/java/org/opensearch/sql/sql/antlr/HighlightTest.java:25]:
```suggestion
    acceptQuery("SELECT HIGHLIGHT('T*') FROM Index WHERE MULTI_MATCH([Tags, Body], 'Time')");
```

File: sql/src/test/java/org/opensearch/sql/sql/parser/AstBuilderTest.java [hunk 3]
```diff
@@ -107,93 +84,59 @@ public void can_build_select_all_and_fields_from_index() {
...
-                "Age_Expr"
-            )
-        ),
-        buildAST("SELECT"
-                + " (age + 10) AS `Age_Expr` "
-                + "FROM test"
-        )
-    );
+            alias("(age + 10)", function("+", qualifiedName("age"), intLiteral(10)), "Age_Expr")),
+        buildAST("SELECT" + " (age + 10) AS `Age_Expr` " + "FROM test"));
```
Comment (GumpacG) [inline sql/src/test/java/org/opensearch/sql/sql/parser/AstBuilderTest.java:111]:
```suggestion
        buildAST("SELECT (age + 10) AS `Age_Expr` FROM test"));
```

File: sql/src/test/java/org/opensearch/sql/sql/parser/context/QuerySpecificationTest.java [hunk 4]
```diff
@@ -119,20 +109,24 @@ void can_collect_sort_options_in_order_by_clause() {
 
   @Test
   void should_skip_sort_items_in_window_function() {
-    assertEquals(1,
-        collect("SELECT name, RANK() OVER(ORDER BY age) "
-            + "FROM test ORDER BY name"
-        ).getOrderByOptions().size());
+    assertEquals(
+        1,
+        collect("SELECT name, RANK() OVER(ORDER BY age) " + "FROM test ORDER BY name")
```
Comment (GumpacG) [inline sql/src/test/java/org/opensearch/sql/sql/parser/context/QuerySpecificationTest.java:114]:
```suggestion
        collect("SELECT name, RANK() OVER(ORDER BY age) FROM test ORDER BY name")
```

**Review by GumpacG — APPROVED** [review]

**Review by Yury-Fridlyand — DISMISSED** [review]

**Review by Yury-Fridlyand — APPROVED** [review]

File: sql/src/main/java/org/opensearch/sql/sql/parser/AstSortBuilder.java (hunk 1 above)
Comment (acarbonetto) [reply sql/src/main/java/org/opensearch/sql/sql/parser/AstSortBuilder.java:57]: You can fix it or leave it.  The change request is unrelated to your work.

**Pull request merged**